AdyenCard/Components/Card/CardViewController.swift

```diff
@@ -46,7 +46,7 @@ internal class CardViewController: FormViewController {
     internal init(configuration: CardComponent.Configuration,
                   shopperInformation: PrefilledShopperInformation?,
                   formStyle: FormComponentStyle,
-                  payment: Payment?,
+                  payment: Payment? = nil,
```

Comment:
let's not use default nil values, because we don't want to miss passing it by mistake :- )

Reply:
👍